templates/gameLog.html

```diff
@@ -76,7 +76,8 @@
                 <td data-bind="text: stl"></td>
                 <td data-bind="text: blk"></td>
                 <td data-bind="text: pf"></td>
-                <td data-bind="text: pts"></td>
+                  <td data-bind="text: pts"></td>
+                  <td data-bind="round: [pts() + 0.4  * fg() - 0.7 * fga() - 0.4*(fta() - ft()) + 0.7 * orb() + 0.3 * (trb() - orb())  + stl() + 0.7 * ast() + 0.7 * blk() - 0.4 * pf() - tov(),0]"></td>
```

Comment:
I'd lean towards a binding.  I guess because its kind of niche and such an inexpensive calculation,  It feels more like something that we calculate as-needed vs calculate and store.  That said, I'm aware that phrases like "feels like" do not generally make compelling technical arguments so I'd be fine with either way.